doc/latex.rst

```diff
@@ -218,6 +218,16 @@ Here are the currently available options together with their default values.
        before or after, but this is accessible currently only by re-defining some
        macros with complicated LaTeX syntax from :file:`sphinx.sty`.
 
+``verbatimhintscontinuation``
+    default ``true``. Boolean to specify if code blocks which happen to be split
```

Comment:
with default to ``false`` we could possibly merge in stable, as it is then only refactoring of non api code, with same output